WeatherWebService/WeatherWebService/CONSTANTS/Constants.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Constants.swift
+//  WeatherWebService
+//
+//  Created by komal lunkad on 30/09/16.
+//  Copyright © 2016 komal lunkad. All rights reserved.
+//
+
+import Foundation
+class Constants {
+    struct StoryBoardID{
+        static let weatherDetailsViewController = "WeatherDetailsViewController"
+    }
+    struct AlertViewMessages{
+        static let alertViewTitle = "Alert"
+        static let whiteSpaceMessage = "Textfield contains only Whitespaces"
+        static let onlyAlphabetMessage = "City name should contain alphabets"
+        static let alertButtonOk = "Ok"
+    }
+    struct ApiDetails {
+        static let url = "http://api.openweathermap.org/data/2.5/weather"
+        static let appID = "82d42d1bbaa0bbec840a96ca44a1660d"
+    }
+    struct ObjectNames {
```

Comment:
change the name to make it more understandable, something like ApiResponseKey would be more understandable